test/MinimalDelegation.execute.t.sol

```diff
@@ -376,26 +362,20 @@ contract MinimalDelegationExecuteTest is TokenHandler, HookHandler, ExecuteFixtu
         (uint256 nonce, uint64 seq) = _buildNextValidNonce(nonceKey);
 
         // Create hash of the calls + nonce and sign it
-        SignedCalls memory signedCalls = SignedCalls({calls: calls, nonce: nonce});
+        SignedCalls memory signedCalls = SignedCallBuilder.init().withCalls(calls).withNonce(nonce);
         bytes32 hashToSign = signerAccount.hashTypedData(signedCalls.hash());
         (uint8 v, bytes32 r, bytes32 s) = vm.sign(signerPrivateKey, hashToSign);
```

Comment:
we should standardize just doing `signerTestKey.sign(...)` for root Key, we have this in a few places